pages/views/styles/samples/multiple_playgrounds.less

```diff
@@ -0,0 +1,12 @@
+.main {
```

Comment:
It looks like it should be moved to `page.less`. 
Because in any case, the styles of this file are applied to all pages and have a higher priority than styles in `page.less`.
In addition, these styles look like they are common for all pages.